auth-starter/src/main/java/com/vaadin/auth/starter/VaadinUserService.java

```diff
@@ -0,0 +1,15 @@
+package com.vaadin.auth.starter;
+
+import org.springframework.security.oauth2.client.oidc.userinfo.OidcUserRequest;
+import org.springframework.security.oauth2.client.oidc.userinfo.OidcUserService;
+import org.springframework.security.oauth2.core.OAuth2AuthenticationException;
+
+public class VaadinUserService extends OidcUserService {
```

Comment:
Should this be a condition bean? I'm seeing myself instantly customizing this to return a "advanced user object" or mingle with the OidcRequest.

Reply:
Absolutely. Consider that both `VaadinUser` and `VaadinUserService` are not yet confirmed and may change or even disappear before the PR is ready for review 👻